include/zephyr/bluetooth/audio/bap.h

```diff
@@ -410,6 +410,9 @@ struct bt_bap_ep_info {
 	 *  otherwise NULL
 	 */
 	struct bt_bap_ep *paired_ep;
+
+	/** Pointer to the preferred QoS settings associated with the endpoint */
+	struct bt_audio_codec_qos_pref *qos_pref;
```

Comment:
```suggestion
	/** Pointer to the preferred QoS settings associated with the endpoint */
	const struct bt_audio_codec_qos_pref *qos_pref;
```
or
```suggestion
	/** Pointer to the preferred QoS settings associated with the endpoint */
	struct bt_audio_codec_qos_pref qos_pref;
```

Suggest to make it const, or make it a value. Returning a non-const pointer to the application may indicate to the application that it can modify it (which would break stuff).